pkg/linguist/linguist_test.go

```diff
@@ -58,6 +56,16 @@ func TestGitAttributes(t *testing.T) {
 	}
 }
 
+func TestIsIgnored(t *testing.T) {
```

Comment:
Could we add a comment about how TestIsIgnored is being used? The words scare me 😅

Reply:
Added a comment. How does that look?